analytics.js

```diff
@@ -1179,13 +1179,16 @@ class Analytics {
       !Array.prototype.find ||
       !Array.prototype.includes ||
       !Promise ||
-      !Object.entries
+      !Object.entries ||
+      !Object.values ||
+      !String.prototype.replaceAll
     ) {
-      ScriptLoader("polyfill", POLYFILL_URL);
+      const id = "polyfill";
+      ScriptLoader(id, POLYFILL_URL);
       const self = this;
       const interval = setInterval(function () {
         // check if the polyfill is loaded
-        if (window.hasOwnProperty("polyfill")) {
+        if (document.getElementById(id) !== null) {
```

Comment:
This is not the right check as the script gets added to the document but it could be still loading.

Reply:
`window.hasOwnProperty("polyfill")` this is returning false even though the polyfill is loaded.
Faced this problem in Chrome version 83 and below. That is the reason for updating the syntax.

Reply:
Addressed.